contacts/index.html

```diff
@@ -0,0 +1,148 @@
+                        <span>Sherman</span>
+                        <span>Leona</span>
+                        <button type="button">X</button>
+                    </div>
+                </li>
+                <li>
+                    <div> 
+                        <button type="button">+</button>
+                        <label>0</label>
+                        <button type="button">-</button>
+                        <span><img src="images/star.png" alt="star icon" width="25" height="25"></span>
+                        <span><img src="images/profile-pics/pic3.jfif" alt="pic3" width="25" height="25"></span>
+                        <input type="checkbox">
+                        <span>Carlson</span>
+                        <span>Annette</span>
+                        <button type="button">X</button>
+                    </div>
+                </li>
+                <li>
+                    <div> 
+                        <button type="button">+</button>
+                        <label>0</label>
+                        <button type="button">-</button>
+                        <span><img src="images/star.png" alt="star icon" width="25" height="25"></span>
+                        <span><img src="images/profile-pics/pic4.jfif" alt="pic4" width="25" height="25"></span>
+                        <input type="checkbox">
+                        <span>Watkins</span>
+                        <span>Ervin</span>
+                        <button type="button">X</button>
+                    </div>
+                </li>
+                <li>
+                    
+                        <button type="button">+</button>
+                        <label>0</label>
+                        <button type="button">-</button>
+                        <span><img src="images/star.png" alt="star icon" width="25" height="25"></span>
+                        <span><img src="images/profile-pics/pic5.jfif" alt="pic5" width="25" height="25"></span>
+                        <input type="checkbox">
+                        <span>Pope</span>
+                        <span>Philip</span>
+                        <button type="button">X</button>
+                    
+                </li>
+
+            </ul>
+
+<!-- ------------------------------------------------------------------------------------------------------ -->
+            <h2>Favs</h2>
+
+            <ol>
+                <li>
+                    work in progress
+                </li>
+                <li>
+                    <p>work in progress</p>
+                </li>
+                <li>
+                    <p>work in progress</p>
+                </li>
+            </ol>
+
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+            <h3>Add new contact</h3>
+
+            <form>
+                <label>First name:</label>
+                <input type="text" name="first_name" value="">
```

Comment:
type="text" is not required on inputs, as they have the default value to type=text anyway